docs/3_development/4_documentation.md

```diff
@@ -29,14 +29,64 @@ Don't forget to update repository settings as described in [Setup GitHub Reposit
 
 ### Deployment
 
-Result of the Docusaurus build step is a static website that can be deployed to
-any of the popular hosting services including GitHub Pages, Render, Netifly or IPFS.
-
-#### Deploy to Render
-
-Even though using GitHub Actions is the easiest option, to this date, they don't have Pull Request previews
-which make it hard to view changes in the browser. Deploying to [Render](https://render.com/) requires
-a one-time setup for which, if you don't have access to ChainSafe's Render account, you can ask your Department Head or DevOps team.
+The result of the Docusaurus build step is a static website that can be deployed to
+any of the popular hosting services including GitHub Pages, CloudFlare or IPFS.
+
+#### Deploy to CloudFlare Pages
+
+One way to achieve this is by deploying their websites on CloudFlare Pages.
+
+CloudFlare Pages is a modern platform for building and deploying websites. It offers several benefits, including:
+
+- High-performance: CloudFlare Pages uses the same global network as CloudFlare's CDN, which means websites load quickly no matter where the visitor is located.
+- Easy to use: Setting up a website on CloudFlare Pages is simple. You can connect your GitHub repository and deploy your site with just a few clicks.
+- Security: CloudFlare Pages offers built-in security features such as HTTPS encryption and DDoS protection.
+
+While CloudFlare workers is a relatively inexpensive option for Continuous Deployment, it still has quotas and requires a monthly subscription. On the other hand, GitHub Actions is free for public repositories so it's a preferred way of deploying websites.
+
```

Comment:
```suggestion
Another reason to use GitHub Actions instead of CloudFlare Workers is the visibility of the deployment process in the PR. CloudFlare Worker following the repository changes will attempt to build and deploy the code in CloudFlare environment and if PR comments for the deployment results are not enabled failure to build or to deploy would be left unnoticed. In contrary, check, build and deploy steps executed in GitHub Actions not only increasing visibility of the whole process but also incorporated into the PR approval process.
```